supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -164,6 +164,53 @@
 
 *See also*:
 
+[discrete]
+[[operator]]
+==== image:images/yes.png[yes] Operator (noun)
+*Description*: In the context of Kubernetes, an Operator is a method of packaging, deploying, and managing a
+Kubernetes application. A Kubernetes application is an application that is both
+deployed on a Kubernetes cluster (including OpenShift clusters) and managed
+using the Kubernetes APIs and `kubectl` or `oc` tooling.
```

Comment:
```suggestion
by the Kubernetes APIs and `kubectl` or `oc` tooling.
```

Suggesting this to remove the 'using'. See the ISG entry for 'using': "To avoid ambiguity, replace this gerund with either "by using" or "that uses"."  In this case, I think 'by' on its own might fix it? If that's not accurate, then it should be 'by using'.

Reply:
I agree, @ndeevy

Reply:
Sure, although the syntactic ambiguity that IBM is trying to avoid only occurs when the object is followed by "using", which isn't the case here because it's a passive construction (and necessarily so). IBM even specifies this:

> To refer to something "using" one thing or another is often ambiguous. For example, the sentence "Back up the server using ABC" might mean "Back up the server by using ABC" or "Back up the server that uses ABC"

So in this example, "is managed using" causes no ambiguity because the object is at the start. Using "by using" here just makes it sound odd and clunky without the object, imo. Compared to "manage an application using" which would indeed need to be "by using".

Reply:
In this case, if there's no ambiguity then it's fair and logical to keep it as is. +1